corehq/ex-submodules/casexml/apps/phone/tests/performance_tests.py

```diff
@@ -21,6 +21,11 @@
 REFERRAL_TYPE = 'referral'
 
 
+def setupModule():
```

Comment:
i also was taking a look into these and got them up and running. i used the `@skip` decorator and for the reasons daniel mentioned: https://github.com/dimagi/commcare-hq/pull/11633/commits/aa84e59a0d8c55106b2e8fa14d0cdcc8b754fb3c#diff-b6d8d6ddbc22ebb9f1fb72a2c5687b30R83